src/components/Api.js

```diff
@@ -11,89 +11,61 @@ export default class Api {
     return Promise.reject(`Error: ${res.status}`);
   }
 
-  async getIntitialCards() {
-    const response = await fetch(this._baseUrl + "/cards", {
+  getInitialCards() {
+    return fetch(`${this._baseUrl}/cards`, {
       headers: this._headers,
-    });
-    return await this._checkResponse(response);
+    }).then(this._checkResponse);
```

Comment:
**Could be improved**

You can make a special method (function ) for fetching and checking responses **not to duplicate** it in every request:

```js

  _request(url, options) {
    return fetch(url, options).then(this._checkResponse)
  }

```

And now you can safely replace `fetch` with `this._request` and remove repeated checking. And you don’t need to change anything else.